docs/requirements.md

```diff
@@ -1,16 +1,59 @@
-# PerfCheck CI Requirements
+# Perf Check CI Requirements
 
 Perf Check CI is a web application to manage jobs that measure and compare performance on another web application.
 
 ## Creating a job
 
-* As a user I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I want to compare relative performance of my branch against master to make sure I'm not shipping code that will slow things down.
+* As a user, I want to have a quick idea how performance on my branch looks, without caring about comparisons against master.
+* As a user, I want to create a job to compare the performance of two branches across multiple request paths.
+* As a user, I know the request I'm working on takes a long time (10-60 seconds), so just want a very rough comparison that takes a smaller amount of time to complete so I can get back to work.
+* As a user, I care about getting a final and accurate-as-possible comparison across two branches.
+* As a user, I want verify that my performance improvements on a branch did not change the response body of the branch.
+
+## Reporting
+
+* As a user, I want to know if the branch performance of each URL is better, about the same, or worse than the reference measurement.
+* As a user, I want to understand how the branch performance thresholds for better/worse are calculated and be able to verify and double check the calculation.
+* As a user, I want to know if URL(s) on my branch still have problematic performance in an absolute way, such as too many database queries, are too long for an acceptable web request, or eats up too much memory.
+
+## Workflow & CI
+
+* As a user, I want to easily start a job from a Github PR without many steps beyond knowing which URL(s) to test.
+* As a user, I want to have a small set of URLs that can be tested against master on most (but not all) of my team's PRs.
+* As a code reviewer, I want to see iterative history that performance was later improved on a PR that initially had poor performance.
```

Comment:
* As a code reviewer, I want to see the history of Perf Checks for a branch so I can verify that the author made performance improvements I requested.

It might be good to be a bit more general here? A reviewer might want to see checks from before the PR was created because: there may be more information and a PR can get re-opened for the same branch. I also think we may want to be careful about suggesting anything about absolute performance in stories. It's possible that a branch already has good performance in the check but it shows that it's doing unexpected queries with caching turned on.

Reply:
This was poorly phrased. I actually meant within a PR. I’ll rethink. 

The specific case is about workflow re: a branch that initially showed performance problems/regressions. The rest of the team is now aware problems exist and would appreciate seeing the iteration towards a more acceptable result. 

It’s not about pre/post PR perf checks or some source of truth about the start of a branch. It’s about opting in to iterative improvement attempts, preferably within the PR. 

Good point on multi-PRs, but not sure we “know” anything about whether we should handle that case yet.

Reply:
> I also think we may want to be careful about suggesting anything about absolute performance in stories. It's possible that a branch already has good performance in the check but it shows that it's doing unexpected queries with caching turned on.

Maybe we can pull this conversation out of this PR. Not entirely sure what you mean here.